components/Bubbles/BubblesGame.js

```diff
@@ -19,6 +19,7 @@ import lever from '../../sprites/lever/leverCharacter';
 import fountain from '../../sprites/fountain/fountainCharacter';
 import fountainLever from '../../sprites/fountainLever/fountainLeverCharacter';
 import canCharacter from '../../sprites/can/canCharacter';
+import {omnivoreUtils as monsterUtils} from '../MatchByColor/omnivoreUtils';
```

Comment:
don't use MatchByColor/omnivoreUtils, that will be deleted. Do not need anymore since the character js file should have the animation index in it.